x/stake/keeper.go

```diff
@@ -2,11 +2,13 @@ package stake
 
```

Comment:
Somewhere in this file (or in a README), can we add a brief outline of the different validator / candidate keyspaces, and what each is used for and/or why the separation makes the staking implementation more efficient?

Reply:
Absolutely, I think I'll add a keeper.md file right here to help describe the stores and subspaces